python/ray/util/client/server/server.py

```diff
@@ -430,9 +430,9 @@ def _prepare_runtime_env(self, job_runtime_env) -> List[str]:
         with disable_client_hook():
             for uri in uris:
                 try:
-                    runtime_env.fetch_package(uri)
-                    print("Adding!: ", runtime_env._get_local_path(uri))
-                    sys.path.insert(0, str(runtime_env._get_local_path(uri)))
+                    pkg_dir = runtime_env.fetch_package(uri)
+                    if pkg_dir:
+                        os.chdir(pkg_dir)
```

Comment:
Here @edoakes

Reply:
LGTM, any reason not to call this `working_dir` as well though?

Reply:
sure